akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -1256,7 +1256,7 @@ final class Flow[In, Out, Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends Gr
    *
    * See also [[Flow.limit]], [[Flow.limitWeighted]]
    */
-  def takeWhile(p: function.Predicate[Out], inclusive: Boolean = false): javadsl.Flow[In, Out, Mat] = new Flow(delegate.takeWhile(p.test, inclusive))
+  def takeWhile(p: function.Predicate[Out], inclusive: Boolean): javadsl.Flow[In, Out, Mat] = new Flow(delegate.takeWhile(p.test, inclusive))
```

Comment:
good catch! 
Could you add an overload that is `def takeWhile(p: function.Predicate[Out]) = takeWhile(p, false)` please?

Reply:
It's already there. In fact, it's been there forever; the two-argument method was later introduced here: https://github.com/akka/akka/pull/20374

Reply:
Ah nice